screens/CheckoutScreen.js

```diff
@@ -31,9 +31,9 @@ export default class CheckoutScreen extends React.Component {
     const productsData = await loadProductsData();
 
     this.setState({
-      customer,
-      currentPoints: customer.points,
-      rewardsAvailable: customer.rewardsAvailable,
+      customer: customer,
+      rewardsAvailable: Math.floor(customer.rewardsAvailable),
+      fullProducts: productsData,
```

Comment:
Nit: doesn't look like you're using this anywhere?

Reply:
i'll remove it!